pints/_optimisers/_nelder_mead.py

```diff
@@ -0,0 +1,337 @@
+#
+# Nelder-Mead simplex method.
+#
+# This file is part of PINTS.
+#  Copyright (c) 2017-2019, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+import numpy as np
+import pints
+
+
+# Flow
```

Comment:
Very nice way of illustrating how the algorithm works!

Reply:
Thanks! It took me a while to fit it into ask-and-tell :-D